dashboard/test/factories.rb

```diff
@@ -324,6 +324,16 @@
     status 'present'
   end
 
+  factory :peer_review do
+    user nil
+    from_instructor false
+    script nil
+    level nil
```

Comment:
It's okay to have these nil in the factory. It just means that there's no default value and whoever is creating these objects is going to have to provide them or the object will fail validation